src/elf/_32/mod.rs

```diff
@@ -12,5 +12,5 @@ pub use self::impure::*;
 
 #[cfg(not(feature = "pure"))]
 mod impure {
-    elf_from_fd!(::std::u32::MAX);
+    elf_from_fd!(!0);
```

Comment:
I don't like this personally because it seems less readable, but since it's going into a macro it's arguably not very readable in the first place.